pom.xml

```diff
@@ -329,6 +329,7 @@
                             <useDefaultExcludes>true</useDefaultExcludes>
                             <excludes>
                                 <exclude>*</exclude>
+                                <exclude>scripts/loader.cmd</exclude>
```

Comment:
remove this. 

Try adding `<cmd>BATCH</cmd>` into the `mappings`

Reply:
Did that and added the configuration to **style.xml** for batch now